tooling/cli/templates/plugin/with-api/examples/tauri-app/src-tauri/Cargo.toml

```diff
@@ -20,9 +20,7 @@ tauri = {{{  tauri_example_dep  }}}
 tauri-plugin-{{ plugin_name }} = { path = "../../../" }
 
 [features]
-# by default Tauri runs in production mode
-# when `tauri dev` runs it is executed with `cargo run --no-default-features` if `devPath` is an URL
-default = [ "custom-protocol" ]
-# this feature is used used for production builds where `devPath` points to the filesystem
+# this feature is used for production builds or when `devPath` points to the filesystem and the built-in dev server is disabled.
+# If you use cargo directly instead of tauri's cli you can use this feature flag to switch between tauri's `dev` and `build` modes.
```

Comment:
- [ ] Maybe add a link to a how-to on this, if it exists?
- [ ] Comment inconsistent with the other Cargo.toml files?

Reply:
> Maybe add a link to a how-to on this, if it exists?

well, i'd expect users to know how to use cargo's features if they use cargo's cli directly tbh.

> Comment inconsistent with the other Cargo.toml files?

is it? The _template_ files should match, and the ones in the repo's example folders should not have a comment at all like it used to be before the PR.

Reply:
Yeah, I know nothing, maybe I shouldn't talk yet 😅